javascript/blaze/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  website: busyhour.id
```

what is this website ? seems more like a commercial platform, rather than a framework documentation
```suggestion
  github: Busy-Hour-Studio/blaze
```